code/game/objects/items/cosmetics.dm

```diff
@@ -49,6 +49,18 @@
 	name = "black lipstick"
 	colour = "black"
 
+/obj/item/lipstick/blue
+	name = "blue lipstick"
+	colour = "blue"
+
+/obj/item/lipstick/green
+	name = "green lipstick"
+	colour = "green"
+
+/obj/item/lipstick/white
+	name = "white lipstick"
+	colour = "white"
```

Comment:
put these before `/obj/item/lipstick/black` so we don't ruin the nice congruity we have between that and `/obj/item/lipstick/black/death`. just cut-paste whatever you've added into line 47 and fix the spacings.